ovs/dal/hybrids/albabackend.py

```diff
@@ -145,9 +145,18 @@ def _usages(self):
         # Collect total usage
         total_size = 0.0
         total_used = 0.0
-        for stats in self.osd_statistics.values():
-            total_size += stats['capacity']
-            total_used += stats['disk_usage']
+        config = Configuration.get_configuration_path(self.abm_cluster.config_location)
+        try:
+            osds_stats = AlbaCLI.run(command='list-osds', config=config)
+        except AlbaError:
+            self._logger.exception('Unable to fetch osd information')
+            return {'free': total_size - total_used,
```

Comment:
Duplicate code, please define `osd_stats = []` before the `try`, then this can be removed